.vs/VSWorkspaceState.json

```diff
@@ -3,7 +3,13 @@
     "",
     "\\API",
     "\\API\\SSW.Consulting",
-    "\\API\\SSW.Consulting\\Controllers"
+    "\\API\\SSW.Consulting.Application",
```

Comment:
.vs folder should be in the .gitignore. Can you add it to the file then use the `git reset --cache -r .` method from last time?

Reply:
same with the `.mfractor` folder